lib/widgets/inset_shadow_box.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
```

Comment:
nit:

```suggestion
import 'package:flutter/widgets.dart';
```

(not sure how much we care about being agnostic about Material though.)